testing/btest/scripts/base/protocols/http/http-lower-case.zeek

```diff
@@ -0,0 +1,11 @@
+# This tests whether the HTTP analyzer handles HTTP keyword in lower case correctly or not.
+#
+# @TEST-EXEC: zeek -C -r $TRACES/http/http-lower-case.pcap
+# @TEST-EXEC: ! test -f dpd.log
+# @TEST-EXEC: ! test -f analyzer.log
+# @TEST-EXEC: grep -q "lowercase_HTTP_keyword" weird.log
```

Comment:
This is a bit unusual. Maybe `zeek-cut ... < weird.log > weird.log.standard and for the other one `zeek-cut ... < weird.log > weird.log.non-standard` and baselining the interesting fields?

Would also make sense to baseline the generated `http.log`.